pkg/config/config_template.yaml

```diff
@@ -895,6 +895,33 @@ api_key:
   #
   # use_device_id_as_hostname: true
 
+  ## @param ping - custom object - optional
+  ## Configure ICMP pings for all hosts in SNMP autodiscovery
+  ## Each device will be pinged with these settings each time the SNMP
+  ## check is run.
+  ##
+  ## By default, we try to use an unprivileged UDP socket to send ICMP
+  ## pings, but some linux systems require using a raw socket.
+  ## See details at the link below:
+  ## https://sturmflut.github.io/linux/ubuntu/2015/01/17/unprivileged-icmp-sockets-on-linux/
```

Comment:
Can we have this info in a DDOG-owned space? Seems a bit odd to link to a personal github page for info.

Reply:
That's a good idea. I think for the sake of getting this in before the freeze I'll just remove the link for now. I'll try to get a page together and link to it before the freeze and open a separate PR to add it.